testdata/public/package/datasources/1.0.0/index.json

```diff
@@ -265,14 +275,25 @@
                 }
               ],
               "dataset": "datasources.examplelog2",
-              "template": "foo: bar\n",
+              "template.path": "stream.yml",
```

Comment:
@mtojek Got stock on this PR as I realised the `.` in the output here will probably cause some troubles in Kibana to read it :-( I still like the naming but need to find a way to make sure we expand it here but still have a short version in the yaml for devs.

Reply:
I'm fine with leaving this as is if it brings more problems that helps ;)